packages/build/webpack/webpack.base.js

```diff
@@ -27,7 +27,9 @@ module.exports = function createConfig() {
       splitChunks: {
         cacheGroups: {
           vendors: {
-            chunks: 'all',
+            // "initial" type only includes statically imported modules in this named "vendor" chunk.
+            // Dynamic imports will be in seperate chunks with their own vendor dependencies included.
```

Comment:
Could you locally uncomment the analyzer lines in webpack.prod.config 
https://github.com/gravitational/webapps/blob/master/packages/build/webpack/webpack.prod.config.js#L25

and examine the chunks visually to see if everything looks as expected? 

(uncomment and then re-build, it will automatically open a browser to examine the chunks)

Reply:
tested and chunks are as expected